WcaOnRails/app/uploaders/avatar_uploader_base.rb

```diff
@@ -3,6 +3,8 @@
 class AvatarUploaderBase < CarrierWave::Uploader::Base
   include CarrierWave::MiniMagick
 
+  DEFAULT_URL = ActionController::Base.helpers.asset_url("missing_avatar_thumb.png", host: ENVied.ROOT_URL).freeze
```

Comment:
Asset url doesn't seem to use `default_url_options` (you can set the host for `asset_url`s separately as you have done before for production). We just specify the host for this particular asset. Other images uses the `assets_host` from the CarrierWave configuration.